models/labels/blue_chip_collection.sql

```diff
@@ -0,0 +1,45 @@
+with nft_trades as (
```

Comment:
this model should be ephemeral?

Reply:
if it needs to be materialized, it should preferably be added to stg folder?

Reply:
ref: [ephemeral](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral)